src/mac/nrn_productsign.sh

```diff
@@ -13,4 +13,5 @@ rm -f $1.signed
 productsign --sign "Developer ID Installer: Michael Hines (8APKV34642)" \
   $1 $1.signed
 
+echo "mv $1.signed $1"
```

Comment:
Using `set -x` in the entire script is probably better (I'm personally in favor of using at least `set -eux`, but that's just a preference).

Reply:
Thanks. That is a better way and I'll try it out.